lib/ox/element.rb

```diff
@@ -150,7 +150,18 @@ def locate(path)
     # _raise_ [NoMethodError] if no match is found
     def method_missing(id, *args, &block)
       has_some = false
-      ids = id.to_s
+      ids = id.to_s.sub('_', ':') # allows easy API with namespaces
+      
+      if ids[-1] == '!' # return array of elements matching ids
```

Comment:
Take a look at the 'each' branch. It included the ```Element#each``` method as well as uses _ as a wildcard for the easy API.